lib/IncomingStreamBridge.js

```diff
@@ -69,6 +69,8 @@ class IncomingStreamBridge extends Emitter
 	 * Get statistics for all tracks in the stream
 	 * 
 	 * See OutgoingStreamTrack.getStats for information about the stats returned by each track.
+	 * 
+	 * @returns {{ [trackId: string]: IncomingStreamTrackBridge.TrackStats }}
```

Comment:
this annotation isn't really needed, the system infers the type from the annotation at `stats` variable two lines below.
if you want to add it, it should at least say the same:

```suggestion
	 * @returns {StreamStats}
```

i generally prefer to avoid repeating stuff, but in this case i think it's okay

Reply:
i copy pasted this from somewhere else (I am not so typescript smart) so we should change it there too.